DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -1486,6 +1486,80 @@ function SetWatchFlags() {
     }
 }
 
+function FailIfCorpseNotHeld(string corpseStr)
```

Comment:
should be able to pass in an actual class, which would also help us detect issues with HX having different classnames, because of compiler errors

```suggestion
function FailIfCorpseNotHeld(class<#var(DeusExPrefix)Carcass> corpseClass)
```